Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.h

```diff
@@ -63,6 +69,14 @@ namespace AZ::Render
         void RenderWireframe(EMotionFX::Mesh* mesh, const AZ::Transform& worldTM, float wireframeScale, float scaleMultiplier,
             const AZ::Color& wireframeColor);
         void RenderJointNames(EMotionFX::ActorInstance* actorInstance, RPI::ViewportContextPtr viewportContext, const AZ::Color& jointNameColor);
+        void RenderNodeOrientations(EMotionFX::ActorInstance* actorInstance, AzFramework::DebugDisplayRequests* debugDisplay, float scale = 1.0f);
+        void RenderLineAxis(
+            AzFramework::DebugDisplayRequests* debugDisplay,
+            AZ::Transform worldTM,      /**< The world space transformation matrix to visualize. */
+            float size,                /**< The size value in units is used to control the scaling of the axis. */
+            bool selected,              /**< Set to true if you want to render the axis using the selection color. */
```

Comment:
nit: I think for these you can just use `//!<` which are easier to maintain (no closing `*/` required)